tests/packagedcode/data/rubygems/gemspec/github.gemspec.expected.json

```diff
@@ -0,0 +1,70 @@
+[
+  {
+    "type": "gem",
+    "namespace": null,
+    "name": "github",
+    "version": "GitHub::VERSION",
+    "qualifiers": {},
+    "subpath": null,
+    "primary_language": "Ruby",
+    "description": "The official `github` command line helper for simplifying your GitHub experience.",
+    "release_date": null,
+    "parties": [
+      {
+        "type": "person",
+        "role": "author",
+        "name": "Chris Wanstrath, Kevin Ballard, Scott Chacon, Dr Nic Williams",
+        "email": "drnicwilliams@gmail.com",
+        "url": null
+      }
+    ],
+    "keywords": [],
+    "homepage_url": "https://github.com/defunkt/github-gem",
+    "download_url": null,
+    "size": null,
+    "sha1": null,
+    "md5": null,
+    "sha256": null,
+    "sha512": null,
+    "bug_tracking_url": null,
+    "code_view_url": null,
+    "vcs_url": null,
+    "copyright": null,
+    "license_expression": null,
+    "declared_license": null,
+    "notice_text": null,
+    "root_path": null,
+    "dependencies": [
+      {
+        "purl": "pkg:gem/rake",
+        "requirement": "",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/rspec",
+        "requirement": "~>1.3.1",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      },
+      {
+        "purl": "pkg:gem/activerecord",
+        "requirement": "~>3.0.0",
+        "scope": "dependencies",
+        "is_runtime": true,
+        "is_optional": false,
+        "is_resolved": false
+      }
+    ],
+    "contains_source_code": null,
```

Comment:
I think that in the general case a Gem contains source code, right? may be not for Pods though